src/rapids_pre_commit_hooks/alpha_spec.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from functools import reduce
+
+import yaml
+from packaging.requirements import Requirement
+from packaging.specifiers import SpecifierSet
+
+from .lint import LintMain
+
+RAPIDS_ALPHA_SPEC_PACKAGES = {
+    "rmm",
+    "pylibcugraphops",
+    "pylibcugraph",
+    "nx-cugraph",
+    "dask-cudf",
+    "cuspatial",
+    "cuproj",
+    "cuml",
+    "cugraph",
+    "cudf",
+    "ptxcompiler",
+    "cubinlinker",
+    "cugraph-dgl",
+    "cugraph-pyg",
+    "cugraph-equivariant",
+    "raft-dask",
+    "pylibwholegraph",
+    "pylibraft",
+    "cuxfilter",
+    "cucim",
+    "ucx-py",
+    "ucxx",
+    "pynvjitlink",
+    "distributed-ucxx",
```

Comment:
We'll have to remember to keep this list up-to-date as we merge the C++ wheel PRs. So far these are the only new ones, but soon we'll have more. I would recommend alphabetizing this list so that it's easier to track.
```suggestion
    "distributed-ucxx",
    "librmm",
    "libucx",
```

Reply:
Added